WordPress/Classes/Models/Blog.m

```diff
@@ -331,10 +331,15 @@ - (NSString *)postFormatTextFromSlug:(NSString *)postFormatSlug
     return formatText;
 }
 
-// WP.COM private blog.
```

Comment:
Would it make sense to add a comment explaining the use of `isPrivate` and how it differs from `isPrivateAtWPCom`?

Reply:
I added some comments to clarify the purpose of both methods.

Btw, I think checking if something is hosted at WP.com is a code smell, and should prolly be replaced with `isAccessibleThroughWPCom`... but  it goes far beyond the scope of this PR.